models/demos/t3000/llama2_70b/tt/llama_common.py

```diff
@@ -176,7 +176,6 @@ def setup_llama_env(llama_version="llama3", batch=32, seq_len=1, n_devices=8, ma
 
     model_config = get_model_config(
         llama_version=llama_version,
-        batch=batch,
```

Comment:
Should batch no longer be an arg for `setup_llama_env`?

Reply:
Thanks, I can remove that